mill

```diff
@@ -1,20 +1,39 @@
 #!/usr/bin/env sh
```

Comment:
Note that this is https://repo1.maven.org/maven2/com/lihaoyi/mill-dist/0.12.14/mill-dist-0.12.14-mill.sh, I didn't update this myself lol.